src/relay/transforms/fuse_ops.cc

```diff
@@ -949,14 +949,29 @@ class FuseMutator : private MixedModeMutator {
 
   Expr MakeNewFunction(GraphPartitioner::Group* group, Type ret_type, Expr body) {
     // If the function has no call, it is not a primitive function.
-    struct HasCallVisitor : ExprVisitor {
+    struct PropertyVisitor : ExprVisitor {
       bool has_call = false;
-      void VisitExpr_(const CallNode* op) final { has_call = true; }
+      bool reshape_only = true;
+      void VisitExpr_(const CallNode* op) final {
+        static const Op& reshape_op_ = Op::Get("reshape");
```

Comment:
similar to my previous comment, we'd at least want to include a check for `reverse_reshape` op, but I think we should generalize and make it less ad hoc if possible